codespell_lib/data/dictionary.txt

```diff
@@ -17281,8 +17281,17 @@ dervie->derive
 dervied->derived
 dervies->derives
 dervived->derived
+desabilities->disabilities
+desability->disability
+desable->disable
```

Comment:
No `desirable` typos out there?

Reply:
`desable->desirable` exists, but is very rare. I'll add it anyways.